api/shape/absolute.js

```diff
@@ -0,0 +1,9 @@
+import Shape from './Shape.js';
+import { makeAbsolute } from '@jsxcad/geometry';
+
+export const absolute = () => (shape) =>
```

Comment:
So this is a new method a.absolute()?

Reply:
Yeah. I only exposed it for testing.

It makes the absolute frame of each object into its local frame.

This might be useful, but I'm not sure exactly what for.

Reply:
Sounds good. My next question was going to be "What is it used for?" 🙂